lxc/console.go

```diff
@@ -237,6 +240,12 @@ func (c *cmdConsole) vga(d lxd.InstanceServer, name string) error {
 	var err error
 	conf := c.global.conf
 
+	// Create a context that is canceled on signal reception.
+	// This is used to enable the function to execute any cleanup defer statements
+	// before exiting.
+	ctx, stop := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM, syscall.SIGHUP)
```

Comment:
I tried with `os.Interrupt` but it does not cancel the underlying ctx. Only `syscall.SIGTERM` seems to work in this case. But I think we should be able to cancel when SIGHUP/SIGINT are fired as this will also have the same effects on the following defer statements.